### PR TITLE
Adds email style quotes to signal blockquotes

### LINKED ISSUE #422: Multi-paragraph blockquote possible?
Hi! I've been using VimWiki in earnest now for a couple of weeks and I love it -- thank you!

One question: is it possible to have a multi-paragraph blockquote? When I place multiple paragraphs of text indented by four spaces, I get *separate* blockquote elements in the resulting HTML. However, other systems (Markdown Here, GitHub Markdown) allow multiple paragraphs to be encased in a single blockquote.

Thanks again!

### PR DESCRIPTION
**Pull request by eltrufas**
fixes #422 

Adds a new function to the html converter to detect blocks of text prefixed with '>' and treats them as  a blockquote.

---

Steps for submitting a pull request:

- [x] **ALL** pull requests should be made against the `dev` branch!
- [x] Take a look at [CONTRIBUTING.MD](https://github.com/vimwiki/vimwiki/blob/dev/CONTRIBUTING.md)
- [x] Reference any related issues.
- [x] Provide a description of the proposed changes.
- [x] PRs must pass Vint tests and add new Vader tests as applicable.
- [x] Make sure to update the documentation in `doc/vimwiki.txt` if applicable,
      including the Changelog and Contributors sections.

### CODE REVIEW

File: doc/vimwiki.txt [hunk 1]
```diff
@@ -1429,6 +1429,14 @@ Text which starts with 4 or more spaces is a blockquote.
     could be styled by CSS in HTML. Blockquotes are usually used to quote a
     long piece of text from another source.
 
+A group of lines prefixed with > also specifies a blockquote, with the caveat
+that the > syntax allow empty lines to signal multiple parragraphs.
```
Comment (Phenitei) [inline doc/vimwiki.txt:1433]: The word "paragraph" takes only one 'r' :)
Thank you for this PR!

Comment (eltrufas) [reply doc/vimwiki.txt]: You're right. Thanks for the review!

**Comment by tinmarino** [comment]
Sorry, I cannot review this one: I don't use html convertion.
But if you add test and they works, I can trust you and just click on "merge" button.

**Comment by tinmarino** [comment]
Sorry I just realize that I was waiting a little for potential changes from you or comments from other.

And I forgot I had to click on Merge button even if I have one of the best Todo, Note taking App.

Excelent PR: should be a model for students

**Pull request merged**